charts/stable/rtorrent-rutorrent/values.yaml

```diff
@@ -88,6 +129,21 @@ persistence:
   passwd:
     enabled: true
     mountPath: "/passwd"
+  exportarr:
+    enabled: true
+    mountPath: "/config"
+    readOnly: true
```

Comment:
```suggestion
```

Reply:
this is not a thing, actually copy what i have given you or return the code back for the persistence, i see no options or code to suggest _your_ exportarr sidecar needs access to the config dir.

your confusing two separate images that have nothing to do with another.

stop.
read the upstreams docs
and fix the chart.

Reply:
This has been corrected.